week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/cart-item-sum.component.ts

```diff
@@ -0,0 +1,9 @@
+import { div } from '../framework/dom-creators';
+import type { Price } from '../types/item';
+
+export function cartItemSum({ value, currency = 'PLN' }: Price): HTMLElement {
```

Comment:
Tutaj pamiętaj że możesz być bardziej "specyficzny" To co jest zwracane to dokładnie `HTMLDivElement` który owszem - dziedziczy po `HTMLElement` - dlatego `TypeScript`'owi to pasuje